### DISCUSSION TITLE
Exception "... already contains entry 'AndroidManifest.xml', cannot overwrite"

### DISCUSSION THREAD

**Discussion by zeptonix** [mapsforge/mapsforge#1277] [Q&A] [answered]
I am new to to the topic "custom osm maps". There I found the mapsforge project. I cloned the project into Android Studio.
Now I have issues compiling the project and I am not sure how to solve it. Now, I hope that somebody can help me.
This Exception occurs:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':mapsforge-samples-android:packageDebug'.
> A failure occurred while executing com.android.build.gradle.tasks.PackageAndroidArtifact$IncrementalSplitterRunnable
   > Zip file 'C:\Users\nicom\AndroidStudioProjects\Mapsforge\mapsforge-samples-android\build\outputs\apk\debug\mapsforge-samples-android-debug.apk' already contains entry 'AndroidManifest.xml', cannot overwrite

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 11s
46 actionable tasks: 13 executed, 33 up-to-date

**Comment by devemux86** [comment, marked as answer]
Using [IntelliJ IDEA](https://www.jetbrains.com/idea/) the project opens / compiles and the samples run successfully.

**Reply by zeptonix** [reply]
My bad, I resolved the problem by changing the path in the gradle file for the android.jar.
Now it is working.
